core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -291,3 +371,17 @@ fun String.fillPlaceholders(vararg strings: String): String {
         filledString = filledString.replaceFirst(keys[i], strings[i])
     return filledString
 }
+
+fun String.getConditionals() = pointyBraceRegex.findAll(this).map { Unique(it.groups[1]!!.value) }.toList()
```

Comment:
See, you _are_ getting the capture here. Am I missing something?

Reply:
Dunno, when I used the `pointyBraceRegex` I always got it including the `<>`, maybe Kotlin regex is just wierd

Reply:
Debug, and Alt-F8 and play around. Or are you coding _everything_ using Vi from a command line? :rofl: 

Regex.findall().first().value is the first match. Regex.findall().first().groups[0].value is, too. Regex.findall().first().groups[1].value is the first capture of the first match... Regex("Hello (\w+) world").findAll("Hello kitty world").first().groups[1]!!.value == "kitty" ... 

Or use one of the offline Regex test playgrounds. What were they named... Is Expresso still open? Coach... http://weitz.de/regex-coach/ ... OK but win-crud... Mint offers github.com/artemanufrij/regextester - hey a YAK! To really get a feeling for Regexes you _really_ need to play with them interactively.

Reply:
Oh, and the only point where kotlin is weird anywhere near Regexes is IMHO raw strings. Those are completely moronic. Try writing a pattern searching for a non-line-end `$`... (answer: `"""${'\'}${'$'}"""`)